### ISSUE TITLE
Behavior score system is broken. 

### ISSUE THREAD

**Issue by Waxz200** [ValveSoftware/Dota2-Gameplay#13380]
### Description

Theres no way to increase your score, no matter what you do. Once you get into this loophole theres no way out. People are literally abusing reports every game, reporting all 9 players. You gonna do something about this shit?.

**Comment by marcelmaster** [comment]
#13340 
same here , upvote please ( thumbs up )

**Comment by immenseforest** [comment]
yes same here upvote valve please fix this, this is ruining new player experience as well as there is no way of recovery once you get put below 6k BS is a loophole the system is broken

**Comment by S1lentInfinity** [comment]
![photo_2023-11-05_03-32-00](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/0f82b19c-4140-4f81-9b32-1652a90bdc76)
![photo_2023-11-05_03-32-17](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/9073db89-1bb3-46ad-981f-f85ea6435645)
![photo_2023-11-06_15-34-51](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/f6779b9c-977a-4384-8831-de013edbe74f)
![photo_2023-11-06_15-35-02](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/55e0f0d0-45b4-4ef8-b49c-bc0ecbb96690)
![2023-11-07_22-41-25](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/04ba48da-624e-456d-9970-552702b0d724)
![2023-11-07_22-41-36](https://github.com/ValveSoftware/Dota2-Gameplay/assets/21246677/c7da641a-c06f-4e0c-831b-d4bd2e736358)
I have the same problem, all I do is complete a series of "road to international" tasks! I don’t do anything that spoils the gameplay of other players, I just play. For the sake of experimentation, I turned on the “do not receive messages” mode and turned off all chats, my decency continued to decline, even when I received 20 likes from 15 games and there were no complaints. There are clearly some problems with the system!

In total I lost over 4.5 thousand in integrity, this is impossible, if there were complaints, I would play single draft every time

**Comment by pajus1337** [comment]
That the changes are needed and it was stupid to make such a system, which was not even checked in the right way I will leave without comment, (I think those responsible for this realize what a failure they have committed...) however, in addition to the amendment itself, the whole scoring system should be reset, because it was completely destroyed by this whole new system

**Comment by cosmonab** [comment]
I am being charged all credit for playing turbo games!!!I get -900 behavior score for turbo games with 13 positive matches and all  matches 
![prikol](https://github.com/ValveSoftware/Dota2-Gameplay/assets/150190069/10982549-cca8-49b3-a775-3c574003da99)

**Comment by cosmonab** [comment]
![sq](https://github.com/ValveSoftware/Dota2-Gameplay/assets/150190069/0d6c2026-ff2f-4b83-8f21-c37e29ecf68a)
gabeeeee pls fixed

**Comment by BledNavalny** [comment]
Agree, but its not Gabe, it some 3-5 guys who in charge and the one who approves, they are just doing social experiments on dota community.
Why we played perfectly since 2011, who asked for your shit system? Dont touch anything, abort august changes( not just these october silent changes)
